basic_client_test.go

```diff
@@ -46,21 +45,21 @@ XhkpT5dliEGFLNe6OOgeWTU1JpEXfCud/GImtNMHQi4EDWQfvWuCNGhOoQ==
 
 	// Now let's write the data to a file to make sure parseTLSAndMacaroon
 	// parses that properly as well.
-	tempDirPath, err := ioutil.TempDir("", ".testCreds")
+	tempDirPath, err := os.MkdirTemp("", ".testCreds")
```

Comment:
Can even use `t.TempDir()`, then the `defer os.RemoveAll(tempDirPath)` is done automatically.

Reply:
Nice! Agree!